src/Eccube/Application.php

```diff
@@ -413,30 +417,30 @@ public function initDoctrine()
             'type' => 'yml',
             'namespace' => 'Eccube\Entity',
             'path' => array(
-                __DIR__ . '/Resource/doctrine',
-                __DIR__ . '/Resource/doctrine/master',
+                __DIR__.'/Resource/doctrine',
+                __DIR__.'/Resource/doctrine/master',
             ),
         );
 
         foreach ($finder as $dir) {
-            $config = Yaml::parse($dir->getRealPath() . '/config.yml');
+            $config = Yaml::parse($dir->getRealPath().'/config.yml');
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'initDoctrine'.